### ISSUE TITLE
"browser compatibility" missing

### ISSUE THREAD

**Issue by johannesnagl** [hyperform/hyperform#19]
would love to see which browsers are fully supported.

IE11 for example, is not supported, in terms of showing a correct timepicker.

**Comment by rodion-arr** [comment]
I also have problems with `<input type="date">` in IE 11: this attribute doesn't work

**Comment by Boldewyn** [comment]
Thanks for the report!

I've added the browser support (as backed by the test matrix in karma.conf.js) to [the README](https://github.com/hyperform/hyperform#browser-support). (Getting the tests running in SauceLabs is actually the main reason for my delayed answer here...)

Unfortunately, UI elements like date pickers are out of scope for Hyperform. I've started another project, [Hyperform UI](https://github.com/hyperform/hyperform-ui), to implement those on top of the excellent jQuery UI widgets. That is in its infancy as of now, but should fit the bill of "provide consistent UI" quite fine.

(jQuery UI? Yes. It's complete, themable, accessible and has wide browser support. Basically all you might want for a UI library. Of course I'd be happy if someone contributed a set of React components, custom elements, Vue.js bindings, ... :wink:)
